corpus/structure/CSIPSTR1/testCase.xsd

```diff
@@ -0,0 +1,1746 @@
+
```

Comment:
Needs to be the XSD document NOT the HTML page

Reply:
Done